src/main/java/de/uni_tuebingen/ub/nppm/db/SaveHelper.java

```diff
@@ -47,6 +50,57 @@ public static List<Map<String, String>> selectAttribute(String table, String att
         return results;
     }
 
+    public static List<Map> selectAttributeMap(String table, String attributeString, int id) throws Exception {
+        List<Map> results = new ArrayList<>();
+
+        try ( Session session = getSession()) {
+            session.getTransaction().begin();
+            String sql = "SELECT " + attributeString + " FROM " + table + " WHERE ID = " + id; // SQL-Abfrage erstellen
+
+            NativeQuery query = session.createNativeQuery(sql);
+
+            List<Object[]> queryResults = query.getResultList(); // Ergebnisse aus der Datenbank als Liste von Object-Arrays
+
+            for (Object[] row : queryResults) {
+                Map<String, Object> rowMap = new HashMap<>();
+
+                // Die Spaltenwerte in die Map einfügen, wobei die Spaltennamen als Schlüssel verwendet werden
+                for (int i = 0; i < row.length; i++) {
+                    String columnName = attributeString.split(",")[i].trim(); // Die Spaltennamen sind mit Kommas getrennt
+                    rowMap.put(columnName, row[i]);
+                }
+
+                results.add(rowMap);
+            }
+
+            session.getTransaction().commit();
+        }
+
+        return results;
+    }
+
+    public static List<Map<String, String>> getMapField(String zieltabelle, int id) throws Exception {
+        List<Map<String, String>> results = new ArrayList<>();
+
+        try ( Session session = getSession()) {
+            session.getTransaction().begin();
+            String sql = "SELECT * FROM " + zieltabelle + " WHERE ID = " + id; // SQL-Abfrage erstellen
+
+            SQLQuery query = session.createSQLQuery(sql);
+            query.setResultTransformer(Criteria.ALIAS_TO_ENTITY_MAP);
+
+            List<Map<String, String>> queryResults = query.list();
+
+            for (Map<String, String> rowMap : queryResults) {
+                results.add(rowMap);
```

Comment:
If you just want to get a result & transform it in a certain way, maybe we should even move these functions to AbstractBase? But i am not sure what the benefit for you is to get this kind of result datatype, so maybe we should discuss this.

Reply:
i moved it to AbstractBase look at created new function in AbstractBase,
i get this kind of result datatype cause i changed in dosave : from : List<Map> attributes = SaveHelper.getMappedList("SELECT " + zielAttribut + " FROM " + zieltabelle + " WHERE " + formularAttribut + "='" + id + "'" + cond);
to:    List<Map<String, String>> attributes = SaveHelper.getAttribute(zieltabelle, zielAttribut, condMap);  it functions perfect of course i could change it maybe back to jsut List<Map> attributes = .... but than i think why ... when it is now like this and maybe it was only working this way .... if you want me to change or try it with just List<Map> attributes then let me know